src/module/account/settings/setting-form.component.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import SettingForm from './setting-form.component';
+
+function renderSettingPage() {
+  render(<SettingForm />);
+}
+
+describe('setting', () => {
+  it('should show a required field warning for each empty input field', async () => {
+    renderSettingPage();
+    const submitButton = screen.getByRole('button', { name: /submit/i });
+    await userEvent.click(submitButton);
+    expect(await screen.findByText(/display name is required/i)).toBeVisible();
+    expect(await screen.findByText(/email is required/i)).toBeVisible();
+    expect(await screen.findByText(/password is required/i)).toBeVisible();
+  });
+
+  it('should show invalid field errors for each invalid input field', async () => {
```

Comment:
please, break this test down into smaller pieces, so far I can see, it tested for different inputs and covered:
- input length
- valid email
- input format

Reply:
Sure. Break tests into max length test, min length test, invalid test